lib/log.rb

```diff
@@ -183,6 +183,21 @@ def reset_dedup
       @dup_buffer.reset if @dup_buffer
     end
 
+    private
+    def sensor(msg)
+      secured_lines = []
+      lines = msg.split("\n")
+      sensor_kw = %W{client_id client_secret subscription_id tenant_id access_key_id secret_access_key secret authorization password oauth token .dockercfg .dockerconfigjson kubeconfig htpasswd ca.crt service-ca.crt tls.crt tls.key service-account.key}
```

Comment:
One more `token-secret.value`,
```
  kind: Secret
  metadata:
    annotations:
      kubernetes.io/service-account.name: deployer
      kubernetes.io/service-account.uid: ed96a7b7-cbc4-4b0f-9b1a-fbdc677a8cdc
      openshift.io/token-secret.name: deployer-token-6j4dk
      openshift.io/token-secret.value: eyJhbGciOiJSUzI1***
```

Perhaps sort them ?

Reply:
+1, the keword `secret` and the updated regexp `/^.*#{kw}.*:.*/` has caught them
```
irb(main):004:0> str="openshift.io/token-secret.value: abcxxxxxxx"
=> "openshift.io/token-secret.value: abcxxxxxxx"
irb(main):005:0> str.match(/^.*secret.*:.*/)
=> #<MatchData "openshift.io/token-secret.value: abcxxxxxxx">
```